NOpenType/Renderer.cs

```diff
@@ -259,22 +259,40 @@ private IEnumerable<GlyphLayout> Layout(int x, int y, string text, TextOptions o
             int scalingFactor = ScalingFactor(options.FontSize);
             int xx = (int)(x * FontToPixelDivisor) / scalingFactor;
             int yy = (int)(y * FontToPixelDivisor) / scalingFactor;
-
-            int drawheightEM = _typeface.Bounds.YMax - _typeface.Bounds.YMin;
-            int lineHeight = (int)(drawheightEM * options.LineHeight);
+            int startXX = xx;
+            
+            // this will need looking at as I don't know if there somethign inside a font that should tell us the real base lineheight, this renders nicely in my tests.
+            int lineHeight = (int)(EmSquare.Size * options.LineHeight);
```

Comment:
yes, there is a settings for this. Check out the specs. You can probably find it in the source here also if you look through the table-classes.